### PR TITLE
Report

### PR DESCRIPTION
**Pull request by vermadeveloper**

### CODE REVIEW

File: application/views/add_photo.php [hunk 1]
```diff
@@ -218,30 +203,21 @@
...
-            angle = cur_angle + angle;
-           }
-       
-       // document.getElementsByClassName('img255').src =php_script_response;
-        $('.fancybox-slide--current .fancybox-content img').attr('alt', angle); 
-        $('.fancybox-slide--current .fancybox-content img').css({'transform': 'rotate(-' + angle + 'deg)'});
-    
+                    var tStamp = +new Date();
+                  
+                    $('body .fancybox-slide--current .fancybox-content .fancybox-image').attr('src','http://localhost/smartpm/assets/job_photo/'+php_script_response+'?t='+ tStamp);
```
Comment (ankur2194) [inline application/views/add_photo.php]: http://localhost/ ????

I know putting media query is another way to solve the reload issue but I suggest to use core JavaScript.
Do not use this method.

Comment (vermadeveloper) [reply application/views/add_photo.php]: Ok i am update this. But i don't think passing a timestamp will create a problem.

Comment (ankur2194) [reply application/views/add_photo.php]: No It will not create any problem.

What about localhost?

Comment (vermadeveloper) [reply application/views/add_photo.php]: Oops. I forgot to replace path.

Comment (ankur2194) [reply application/views/add_photo.php]: ok change it.

Comment (vermadeveloper) [reply application/views/add_photo.php]: Done

File: application/views/add_photo.php [hunk 2]
```diff
@@ -205,10 +205,11 @@
                    // location.reload();
                     var tStamp = +new Date();
                   
-                    $('body .fancybox-slide--current .fancybox-content .fancybox-image').attr('src','http://localhost/smartpm/assets/job_photo/'+php_script_response+'?t='+ tStamp);
-                    src='http://localhost/smartpm/assets/job_photo/'+php_script_response;
+                    $('body .fancybox-slide--current .fancybox-content .fancybox-image').attr('src',baseUrl+'assets/job_photo/'+php_script_response+'?t='+ tStamp);
+                    src=baseUrl+'assets/job_photo/'+php_script_response;
+                    alert(src);
```
Comment (ankur2194) [inline application/views/add_photo.php]: Do you need this alert on site?

**Pull request merged**